docs/Reference/Alternative-EC-Curves.md

```diff
@@ -0,0 +1,31 @@
+description: Using alternative Elliptic Curves in Besu
+<!--- END of page meta data -->
+
+# Using alternative Elliptic Curves in Besu
+
+!!! caution
+    Alternative Elliptic Curves is an experimental feature.
+
+By default, Besu uses the Ethereum standard secp256k1 elliptic curve (EC).
+However, when running nodes in a private network, it is possible to configure an alternative elliptic curve.
+
+The configuration for what elliptic curve Besu will use is done in the network configuration section of genesis file, using the [`ecCurve`](Config-Items.md#Configuration_Items) key:
+
+```bash
+{
+  "genesis": {
+    "config": {
+      "ecCurve": "secp256k1",
+    [...]
+  },
+  [...]
+}
+```
+
+!!! attention
+    All nodes in the network **MUST** use the same elliptic curve.
+
+The following Elliptic Curves are supported by Besu:
```

Comment:
Could provide some hints about the use case for which we should use one or the other?

Reply:
At the moment the only reason is that secp256r1 is NIST approved and sepc256k1 is not. But this is just the first step towards a compliant node so I don't really think we should mention it in the docs for now.

I'd say that the users interested in using alternative keys will know why they need to do so.